### ISSUE TITLE
AMS setup fails due to header error of it.

### ISSUE THREAD

**Issue by ryzom-pipeline** [ryzom/ryzomcore#193]
**[Original report](https://bitbucket.org/ryzom/ryzomcore/issue/193) by Dylan Geels (Bitbucket: [Geels](https://bitbucket.org/Geels), GitHub: [Geels](https://github.com/Geels)).**

----------------------------------------

The error is shown after the tool step.

```
#!

Warning: session_start(): Cannot send session cache limiter - headers already sent (output started at /srv/ryzomcore/code/web/public_php/setup/header.php:31) in /srv/ryzomcore/code/web/public_php/admin/functions_auth.php on line 82
Warning: Cannot modify header information - headers already sent by (output started at /srv/ryzomcore/code/web/public_php/setup/header.php:31) in /srv/ryzomcore/code/web/public_php/admin/functions_auth.php on line 84
Warning: Cannot modify header information - headers already sent by (output started at /srv/ryzomcore/code/web/public_php/setup/header.php:31) in /srv/ryzomcore/code/web/public_php/admin/functions_auth.php on line 85
Warning: Cannot modify header information - headers already sent by (output started at /srv/ryzomcore/code/web/p ublic_php/setup/header.php:31) in /srv/ryzomcore/code/web/public_php/admin/functions_auth.php on line 86
Warning: Cannot modify header information - headers already sent by (output started at /srv/ryzomcore/code/web/public_php/setup/header.php:31) in /srv/ryzomcore/code/web/public_php/admin/functions_auth.php on line 87
Warning: Cannot modify header information - headers already sent by (output started at /srv/ryzomcore/code/web/public_php/setup/header.php:31) in /srv/ryzomcore/code/web/public_php/admin/functions_auth.php on line 88 .
```

**Comment by ryzom-pipeline** [comment]
**Original comment by Jan Boon (Bitbucket: [Jan Boon](https://bitbucket.org/Jan Boon), ).**

----------------------------------------

This should fix #193

**Comment by ryzom-pipeline** [comment]
**Original comment by Dylan Geels (Bitbucket: [Geels](https://bitbucket.org/Geels), GitHub: [Geels](https://github.com/Geels)).**

----------------------------------------

Okay the error is not showing in the installer but still no role_support is from it.

**Comment by ryzom-pipeline** [comment]
**Original comment by Jan Boon (Bitbucket: [Jan Boon](https://bitbucket.org/Jan Boon), ).**

----------------------------------------

This might fix #193
